executor/prepared_test.go

```diff
@@ -1202,7 +1202,7 @@ func TestPrepareStmtAfterIsolationReadChange(t *testing.T) {
 	ps = []*util.ProcessInfo{tkProcess}
 	tk.Session().SetSessionManager(&testutil.MockSessionManager{PS: ps})
 	rows = tk.MustQuery(fmt.Sprintf("explain for connection %d", tkProcess.ID)).Rows()
-	require.Equal(t, rows[len(rows)-1][2], "cop[tiflash]")
+	require.Equal(t, rows[len(rows)-1][2], "mpp[tiflash]")
 
```

Comment:
function `TestPrepareStmtAfterIsolationReadChange` test failed because time out failure:
```
[2022/08/28 21:13:45.400 -07:00] [INFO] [batch_coprocessor.go:587] ["retry for TiFlash peer with region missing"] ["region id"=2]
[2022/08/28 21:13:47.230 -07:00] [INFO] [batch_coprocessor.go:587] ["retry for TiFlash peer with region missing"] ["region id"=2]
[2022/08/28 21:13:48.234 -07:00] [INFO] [batch_coprocessor.go:587] ["retry for TiFlash peer with region missing"] ["region id"=2]
[2022/08/28 21:13:50.140 -07:00] [INFO] [batch_coprocessor.go:587] ["retry for TiFlash peer with region missing"] ["region id"=2]
[2022/08/28 21:13:51.811 -07:00] [INFO] [batch_coprocessor.go:587] ["retry for TiFlash peer with region missing"] ["region id"=2]
[2022/08/28 21:13:53.485 -07:00] [INFO] [batch_coprocessor.go:587] ["retry for TiFlash peer with region missing"] ["region id"=2]
[2022/08/28 21:13:53.485 -07:00] [WARN] [backoff.go:158] ["tiflashRPC backoffer.maxSleep 10000ms is exceeded, errors:\nCannot find region with TiFlash peer at 2022-08-28T21:13:48.23477-07:00\nCannot find region with TiFlash peer at 2022-08-28T21:13:50.140297-07:00\nCannot find region with TiFlash peer at 2022-08-28T21:13:51.811599-07:00\nlongest sleep type: tiflashRPC, time: 10659ms"]
[2022/08/28 21:13:53.485 -07:00] [INFO] [tidb.go:271] ["rollbackTxn called due to ddl/autocommit failure"]
[2022/08/28 21:13:53.485 -07:00] [WARN] [session.go:2066] ["run statement failed"] [schemaVersion=34] [error="[tikv:9012]TiFlash server timeout"] [errorVerbose="[tikv:9012]TiFlash server timeout\ngithub.com/pingcap/errors.AddStack\n\t/Users/hyuan/go/pkg/mod/github.com/pingcap/errors@v0.11.5-0.20211224045212-9687c2b0f87c/errors.go:174\ngithub.com/pingcap/errors.Trace\n\t/Users/hyuan/go/pkg/mod/github.com/pingcap/errors@v0.11.5-0.20211224045212-9687c2b0f87c/juju_adaptor.go:15\ngithub.com/pingcap/tidb/store/copr.buildBatchCopTasksCore\n\t/Users/hyuan/tidb/store/copr/batch_coprocessor.go:611\ngithub.com/pingcap/tidb/store/copr.buildBatchCopTasksForNonPartitionedTable\n\t/Users/hyuan/tidb/store/copr/batch_coprocessor.go:530\ngithub.com/pingcap/tidb/store/copr.(*MPPClient).ConstructMPPTasks\n\t/Users/hyuan/tidb/store/copr/mpp.go:83\ngithub.com/pingcap/tidb/planner/core.(*mppTaskGenerator).constructMPPTasksImpl\n\t/Users/hyuan/tidb/planner/core/fragment.go:347\ngithub.com/pingcap/tidb/planner/core.(*mppTaskGenerator).generateMPPTasksForFragment\n\t/Users/hyuan/tidb/planner/core/fragment.go:252\ngithub.com/pingcap/tidb/planner/core.(*mppTaskGenerator).generateMPPTasksForExchangeSender\n\t/Users/hyuan/tidb/planner/core/fragment.go:233\ngithub.com/pingcap/tidb/planner/core.(*mppTaskGenerator).generateMPPTasks\n\t/Users/hyuan/tidb/planner/core/fragment.go:82\ngithub.com/pingcap/tidb/planner/core.GenerateRootMPPTasks\n\t/Users/hyuan/tidb/planner/core/fragment.go:73\ngithub.com/pingcap/tidb/executor.(*MPPGather).Open\n\t/Users/hyuan/tidb/executor/mpp_gather.go:112\ngithub.com/pingcap/tidb/executor.(*ExecStmt).openExecutor\n\t/Users/hyuan/tidb/executor/adapter.go:909\ngithub.com/pingcap/tidb/executor.(*ExecStmt).Exec\n\t/Users/hyuan/tidb/executor/adapter.go:468\ngithub.com/pingcap/tidb/session.runStmt\n\t/Users/hyuan/tidb/session/session.go:2197\ngithub.com/pingcap/tidb/session.(*session).ExecuteStmt\n\t/Users/hyuan/tidb/session/session.go:2061\ngithub.com/pingcap/tidb/testkit.(*TestKit).ExecWithContext\n\t/Users/hyuan/tidb/testkit/testkit.go:276\ngithub.com/pingcap/tidb/testkit.(*TestKit).MustExecWithContext\n\t/Users/hyuan/tidb/testkit/testkit.go:111\ngithub.com/pingcap/tidb/testkit.(*TestKit).MustExec\n\t/Users/hyuan/tidb/testkit/testkit.go:106\ngithub.com/pingcap/tidb/executor_test.TestPrepareStmtAfterIsolationReadChange\n\t/Users/hyuan/tidb/executor/prepared_test.go:1200\ntesting.tRunner\n\t/opt/homebrew/Cellar/go/1.18.1/libexec/src/testing/testing.go:1439\nruntime.goexit\n\t/opt/homebrew/Cellar/go/1.18.1/libexec/src/runtime/asm_arm64.s:1259"] [session="{\n  \"currDBName\": \"test\",\n  \"id\": 1,\n  \"preparedStmtCount\": 1,\n  \"status\": 2,\n  \"strictMode\": true,\n  \"user\": {\n    \"Username\": \"root\",\n    \"Hostname\": \"localhost\",\n    \"CurrentUser\": true,\n    \"AuthUsername\": \"root\",\n    \"AuthHostname\": \"%\"\n  }\n}"]
    testkit.go:113: 
        	Error Trace:	/Users/hyuan/tidb/executor/testkit.go:113
        	            				/Users/hyuan/tidb/executor/testkit.go:106
        	            				/Users/hyuan/tidb/executor/prepared_test.go:1200
        	Error:      	Received unexpected error:
        	            	[tikv:9012]TiFlash server timeout
        	            	github.com/pingcap/errors.AddStack
        	            		/Users/hyuan/go/pkg/mod/github.com/pingcap/errors@v0.11.5-0.20211224045212-9687c2b0f87c/errors.go:174
        	            	github.com/pingcap/errors.Trace
        	            		/Users/hyuan/go/pkg/mod/github.com/pingcap/errors@v0.11.5-0.20211224045212-9687c2b0f87c/juju_adaptor.go:15
        	            	github.com/pingcap/tidb/store/copr.buildBatchCopTasksCore
        	            		/Users/hyuan/tidb/store/copr/batch_coprocessor.go:611
        	            	github.com/pingcap/tidb/store/copr.buildBatchCopTasksForNonPartitionedTable
        	            		/Users/hyuan/tidb/store/copr/batch_coprocessor.go:530
        	            	github.com/pingcap/tidb/store/copr.(*MPPClient).ConstructMPPTasks
        	            		/Users/hyuan/tidb/store/copr/mpp.go:83
        	            	github.com/pingcap/tidb/planner/core.(*mppTaskGenerator).constructMPPTasksImpl
        	            		/Users/hyuan/tidb/planner/core/fragment.go:347
        	            	github.com/pingcap/tidb/planner/core.(*mppTaskGenerator).generateMPPTasksForFragment
        	            		/Users/hyuan/tidb/planner/core/fragment.go:252
        	            	github.com/pingcap/tidb/planner/core.(*mppTaskGenerator).generateMPPTasksForExchangeSender
        	            		/Users/hyuan/tidb/planner/core/fragment.go:233
        	            	github.com/pingcap/tidb/planner/core.(*mppTaskGenerator).generateMPPTasks
```